codegen/tests/simple/servicehandler.go

```diff
@@ -35,10 +42,32 @@ func NewServiceHandler(genCallback GenCallback, serviceInterface *ServiceInterfa
 	return &ServiceHandler{genCallback, serviceInterface}
 }
 
+// Handler Error
+func (s *ServiceHandler) handleError(ctx context.Context, w http.ResponseWriter, kind common.Kind, message string, cause error) {
```

Comment:
This function doesn't need to be generated because it doesn't contain any custom code. The following code could replace `common.HandleError`:

```go
func HandleError(ctx context.Context, w http.ResponseWriter, kind Kind, message string, cause error,
		httpErrorMapper func(context.Context, ServerError) *HTTPError) {

	logEntry := GetLogEntryFromContext(ctx)
	logEntry.Error(cause)

	err := CreateError(ctx, kind, message, cause)
	httpError := httpErrorMapper(ctx, err)
	if httpError == nil {
		switch t := cause.(type) {
		case CustomError:
			e := t.HTTPError()
			httpError = &e
		default:
			e := mapError(ctx, &err)
			httpError = &e
		}
	}
	httpError.WriteError(ctx, w)
}

func mapError(ctx context.Context, err error) HTTPError {

	var (
		httpCode        int
		errorCode, desc string
	)
```